src/Helper/PaymentHelper.php

```diff
@@ -26,18 +26,46 @@ public function __construct(PaymentMethodRepositoryContract $paymentMethodReposi
 		$this->paymentMethodRepository = $paymentMethodRepository;
 	}
 
-	/**
-	 * @return int|null
-	 */
-	public function getPaymentMethodId()
```

Comment:
was this function not used anywhere so far?

Reply:
`getPaymentMethodId` in `PaymentHelper` was also used before to get the payment method id of etsy direct checkout payment method. because of that I added this payment method as default input for the function.

`getPaymentMethodId` in `OrderHelper` was only used at 2 places.
I don't understand why there are 2 funtions for "getPaymentMethodId" :D 
So I just merged them to one function